tedge/src/cli/disconnect/cli.rs

```diff
@@ -0,0 +1,36 @@
+use crate::{
+    cli::connect::{az::AZURE_CONFIG_FILENAME, c8y::C8Y_CONFIG_FILENAME},
+    command::{BuildCommand, BuildContext, Command},
+};
+
+use super::bridge::DisconnectBridge;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+pub enum TedgeDisconnectBridgeCli {
+    /// Remove bridge connection to Cumulocity.
+    C8y,
+    /// Remove bridge connection to Azure.
+    Az,
+}
+
+impl BuildCommand for TedgeDisconnectBridgeCli {
+    fn build_command(
+        self,
+        _context: BuildContext,
+    ) -> Result<Box<dyn Command>, crate::config::ConfigError> {
+        let cmd = match self {
+            TedgeDisconnectBridgeCli::C8y => DisconnectBridge {
+                config_file: C8Y_CONFIG_FILENAME.into(),
```

Comment:
I would build here the full path of the bridge config

Reply:
I wasn't going to touch the existing implementations.